app/controllers/v1/user_token_controller.rb

```diff
@@ -0,0 +1,16 @@
+module V1
+  class UserTokenController < Knock::AuthTokenController
+    private
+
+    def auth_params
+      token = TokenRequest.new(request)
+
+      # NOTE: Knock wants you to get your token using a username and password
+      # from a post body. This is bad from a security standpoint. Also, to avoid
+      # monkey patching a lot of places, we just return a hash that has the
+      # :email and :password keys, even though they're really the :api_key and
+      # :unencrypted_api_secret
```

Comment:
edit the line about api_key and unencrypted_api_secret. we call the column password now